samples/tacos/src/main/kotlin/com/slack/circuit/tacos/ui/theme/Theme.kt

```diff
@@ -74,13 +83,67 @@ private val DarkColors =
     scrim = md_theme_dark_scrim,
   )
 
+private val LightTacoColors =
+  TacoColorScheme(
+    primaryBottomBarContainer = tc_light_bottomBarRoseContainer,
+    secondaryBottomBarContainer = tc_light_bottomBarRedContainer,
+    onPrimaryBottomBarContainer = tc_light_onBottomBarRoseContainer,
+    onSecondaryBottomBarContainer = tc_light_onBottomBarRedContainer,
+  )
+
+private val DarkTacoColors =
+  TacoColorScheme(
+    primaryBottomBarContainer = tc_dark_bottomBarRoseContainer,
+    secondaryBottomBarContainer = tc_dark_bottomBarRedContainer,
+    onPrimaryBottomBarContainer = tc_dark_onBottomBarRoseContainer,
+    onSecondaryBottomBarContainer = tc_dark_onBottomBarRedContainer,
+  )
+
 @Composable
-fun TacoTheme(useDarkTheme: Boolean = isSystemInDarkTheme(), content: @Composable () -> Unit) {
-  val colors =
+fun TacoTheme(
+  useDarkTheme: Boolean = isSystemInDarkTheme(),
+  content: @Composable () -> Unit
+) {
+  val materialColors =
+    when {
+      useDarkTheme -> DarkMaterialColors
+      else -> LightMaterialColors
+    }
+  val tacoColors =
     when {
-      useDarkTheme -> DarkColors
-      else -> LightColors
+      useDarkTheme -> DarkTacoColors
+      else -> LightTacoColors
     }
 
-  MaterialTheme(colorScheme = colors, content = content)
+  MaterialTheme(colorScheme = materialColors) {
+    CompositionLocalProvider(LocalTacoColorScheme provides tacoColors, content = content)
+  }
 }
+
+internal val LocalTacoColorScheme = staticCompositionLocalOf { LightTacoColors }
+
+object TacoTheme {
+  val colorScheme: TacoColorScheme
+    @Composable
+    @ReadOnlyComposable
+    get() = LocalTacoColorScheme.current
+}
```

Comment:
I took inspiration from `MaterialTheme` here, and set up `TacoTheme`. Doing so allows me to better support light vs dark without having to add conditionals to the code